tests/features/tmgmt/tmgmt_poetry_translations.feature

```diff
@@ -39,13 +42,72 @@ Feature: TMGMT Poetry features
     Then I should see the error message "There was an error with the Poetry request."
     And I should see "Rejected" in the "French" row
     And I should see "Rejected" in the "Italian" row
+    Then I am logged in as a user with the "cem" role
     When I go to "/admin/config/regional/tmgmt_translator/manage/tmgmt_poetry_test_translator"
```

Comment:
Actually i noticed this makes no difference, sometimes people use '/' sometimes not. Which do you prefer?